backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitChannelPoolImpl.java

```diff
@@ -89,6 +91,11 @@ public RabbitChannelPoolImpl(RabbitMQConnectionFactory factory) {
         pool = new GenericObjectPool<>(pooledChannelsFactory);
     }
 
+    @Override
+    public Flux<AcknowledgableDelivery> receive(String queueName) {
+        return null;
```

Comment:
Why null?

Reply:
Maybe we could explicify this?

Reply:
To be honest, I will rework it for many reasons but it's not for today